packs/elastic/README.md

```diff
@@ -0,0 +1,170 @@
+# Elasticsearch integration pack
+
+Pack provides many operations helping to manage your Elasticsearch indices. Current functionality is mostly based on [curator](http://www.elastic.co/guide/en/elasticsearch/client/curator/current/).
+
+## Features
+
+Pack provides a set of actions based on curator.
+
+## Curator based actions
+
+Action | Description
+------ | -----------
+**alias** | Add indices to or remove them from aliases.
+**allocation** | Set routing allocation based on tags.
+**bloom** | Disable the bloom filter cache for indices.
+**close** | Close indices.
+**delete.indices** | Delete indices.
+**delete.snapshots** | Delete snapshots.
+**open** | Open indices.
```

Comment:
Yeap so it will be *delete.document* or something with another entry point. Here grouping is made for curator stuff..
> ... some other packs already follow this convention - st2.kv.get, st2.actions.list, etc.).

A bit different, under `kv.*` for example they share the same interface, in case of elastic it happened in a different way.

> And using resource as a prefix means you ...

Yes, then it was worth making smth like *elastic.curator.delete.indices* - which looks too **bloated**, I guess.
If I want  cleaner way to invoke actions, it will make the way of finding particular one a bit more intricate.

If 4-level actions seem sane then it makes sense to group like you suggest. I totally agree with you, but all those you mention were 3-level actions. What do you think?